app.js

```diff
@@ -0,0 +1,10 @@
+import instruments from '..data/instruments.js';
+import renderInstruments from './render-instruments.js';
+
+const instList = document.getElementById('instruments');
+
+for (let i = 0; i < instruments.length; i++) {
+    const instrument = instruments[i];
+    const dom = renderInstruments(instrument);
+    instList.appendChild(dom);
+}
```

Comment:
You have this exact same code in products.js... why?

Reply:
I meant to go back and check the assignment - I thought they were requiring that it be called products.js and I was going to fix it. But I forgot that step!